client/components/Reviews.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Review from './Review.jsx';
+
+function Reviews(props) {
+  // console.log(props.reviews)
+  return (
+    <div>
+      {props.reviews.map( (review) => {
+        return <Review key={review._id} review={review} />;
```

Comment:
Nice, adding the _id generated by the database as the unique key is a good idea.